vermouth/data/force_fields/martini30dev/aminoacids.ff

```diff
@@ -0,0 +1,859 @@
+prot_default_bb_type P2
+stiff_fc 1000000
+
+;;; GLYCINE
+
+[ moleculetype ]
+; molname       nrexcl
+GLY                1			
+
+[ atoms ]
+;id  type resnr residu atom cgnr   charge
+ 1   SP1   1     GLY    BB     1      0      
+
+;;; ALANINE
+
+[ moleculetype ]
+; molname       nrexcl
+ALA                1
+
+[ atoms ]
+;id type resnr residu atom cgnr   charge
+ 1   P1    1     ALA    BB     1      0
+
+;;; CYSTEINE
+
+[ moleculetype ]
+; molname       nrexcl
+CYS                1
+
+[ atoms ]
+;id type resnr residu atom cgnr   charge
+ 1   $prot_default_bb_type     1   CYS    BB     1      0
+ 2   TC4    1   CYS    SC1    2      0     
+
+[bonds]
+#meta {"group": "Side chain bonds"}
+;  i     j   funct   length  force.c.
+  BB   SC1    1       0.341     7500        
+
+;;; VALINE
+
+[ moleculetype ]
+; molname       nrexcl
+VAL                1
+
+[ atoms ]
+;id type resnr residu atom cgnr   charge
+ 1   $prot_default_bb_type   1     VAL     BB     1      0    
+ 2   SC2  1     VAL     SC1    2      0    
+
+[ constraints ]
+#meta {"group": "Side chain bonds", "ifndef": "FLEXIBLE"}
+;  i     j   funct   length  
+  BB   SC1    1       0.292
+
+[ bonds ]
+#meta {"group": "Side chain bonds", "ifdef": "FLEXIBLE"}
+;  i     j   funct   length  
+  BB   SC1    1       0.292  $stiff_fc
+
+;;; LEUCINE
+
+[ moleculetype ]
+; molname       nrexcl
+LEU                1
+
+[ atoms ]
+;id type resnr residu atom cgnr   charge
+ 1   $prot_default_bb_type   1     LEU     BB     1      0    
+ 2   SC1  1     LEU     SC1    2      0    
+
+[bonds]
+#meta {"group": "Side chain bonds"}
+;  i     j   funct   length  force.c.
+  BB   SC1    1       0.363     7500    
+
+;;; ISOLEUCINE
+
+[ moleculetype ]
+; molname       nrexcl
+ILE                1
+
+[ atoms ]
+;id type resnr residu atom cgnr   charge
+ 1   $prot_default_bb_type   1     ILE     BB     1      0    
+ 2   SC1  1     ILE     SC1    2      0    
+
+[ constraints ]
+#meta {"group": "Side chain bonds", "ifndef": "FLEXIBLE"}
+;  i     j   funct   length  
+  BB   SC1    1       0.341
+
+[ bonds ]
+#meta {"group": "Side chain bonds", "ifdef": "FLEXIBLE"}
+;  i     j   funct   length  
+  BB   SC1    1       0.341  $stiff_fc
+
+;;; METHIONINE
+
+[ moleculetype ]
+; molname       nrexcl
+MET                1
+
+[ atoms ]
+;id type resnr residu atom cgnr   charge
+ 1   $prot_default_bb_type   1     MET     BB     1      0    
+ 2   C4   1     MET     SC1    2      0     
+
+[bonds]
+#meta {"group": "Side chain bonds"}
+;  i     j   funct   length  force.c.
+  BB   SC1    1       0.40     2500
+
+;;; PROLINE
+
+[ moleculetype ]
+; molname       nrexcl
+PRO                1
+
+[ atoms ]
+;id type resnr residu atom cgnr   charge
+ 1   SP1a   1     PRO     BB     1      0    
+ 2   SC3    1     PRO     SC1    2      0    
+
+[bonds]
+#meta {"group": "Side chain bonds"}
+;  i     j   funct   length  force.c.
+  BB   SC1   1       0.333     7500
+
+[ moleculetype ]
+; molname       nrexcl
+HYP                1
```

Comment:
HYP?

Reply:
Hydroxy-proline. Very common in collagen.